src/routes/locking/entities/holder.page.entity.ts

```diff
@@ -0,0 +1,8 @@
+import { Page } from '@/routes/common/entities/page.entity';
+import { Holder } from '@/routes/locking/entities/holder.entity';
+import { ApiProperty } from '@nestjs/swagger';
+
+export class HolderPage extends Page<Holder> {
```

Comment:
We'd need to rename this if you agree with the above as well.

Reply:
Changed in [d103261](https://github.com/safe-global/safe-client-gateway/pull/1557/commits/d103261dc2f90845eadbf8913523842a099ce968)